### DISCUSSION TITLE
请问目前1.0.0版本对应的是哪个分支啊？我只看到dev1.0.0分支？但是dev是开发啊，不太自信

### DISCUSSION THREAD

**Discussion by stgztsw** [apache/doris#10363] [A - General / Q&A]
请问目前1.0.0版本对应的是哪个分支啊？我只看到dev1.0.0分支？但是dev是开发啊，不太自信

**Comment by dataalive** [comment]
是在分支发布，比如dev1.0.1会发布为1.1.0。

**Comment by kpfly** [comment]
version 1.1 will be released in the next 1-3 days , maybe you can try this
version.

dataalive ***@***.***> 于2022年6月26日周日 12:25写道：

**Comment by hf200012** [comment]
> 请问目前1.0.0版本对应的是哪个分支啊？我只看到dev1.0.0分支？但是dev是开发啊，不太自信

就是这个dev-1.0.0
